### PR TITLE
Updates README with steps how to get keys

### PR DESCRIPTION
**Pull request by mPlagge**

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -24,7 +24,14 @@ Tools are configured via:
 
 ## Tool config
 
-This is the tool config format. It is shown in /config/tool_config.template.json.
+#### Optaining keys
```
Comment (vliedel) [inline tools/README.md]:
```suggestion
#### Obtaining keys
```

File: tools/README.md [hunk 2]
```diff
@@ -24,7 +24,14 @@ Tools are configured via:
...
+
+Go to [my.crownstone](my.crownstone.rocks) and copy the access token to your clipboard.
+Click of the [Crownstone REST API explorer](https://my.crownstone.rocks/explorer/), and set your access tokes.
+Now you are able to make call to the API.
```
Comment (vliedel) [inline tools/README.md]:
```suggestion
Now you are able to make calls to the API.
```

File: tools/README.md [hunk 3]
```diff
@@ -24,7 +24,14 @@ Tools are configured via:
...
+
+To get your sphere keys to to the *users/me* tab, click the *try it out* button, and copy your ID from the responce body.
```
Comment (vliedel) [inline tools/README.md]:
```suggestion
To get your sphere keys go to the *users/me* tab, click the *try it out* button, and copy your ID from the response body.
```

File: tools/README.md [hunk 4]
```diff
@@ -24,7 +24,14 @@ Tools are configured via:
...
+With this ID you can access your keys on the */users/{id}/keysV2* tab. Fill your ID into the form and the responce body should display the keys and keytypes. These have to be formatted like shown bellow. This is the tool config format. It is shown in /config/tool_config.template.json.
```
Comment (vliedel) [inline tools/README.md]:
```suggestion
With this ID you can access your keys on the */users/{id}/keysV2* tab. Fill your ID into the form and the response body should display the keys and key types. These have to be formatted like shown bellow.
```

File: tools/README.md [hunk 5]
```diff
@@ -24,7 +24,14 @@ Tools are configured via:
 
 ## Tool config
```
Comment (vliedel) [inline tools/README.md:26]:
```suggestion

This is the tool config format. It is shown in /config/tool_config.template.json.

```

**Review by vliedel — CHANGES_REQUESTED** [review]

**Comment by mPlagge** [comment]
Added feedback to README

**Review by mPlagge — COMMENTED** [review]
Good suggestions

**Pull request merged**